src/wp-includes/rest-api/endpoints/class-wp-rest-autosaves-controller.php

```diff
@@ -390,11 +407,26 @@ public function create_post_autosave( $post_data ) {
 			do_action( 'wp_creating_autosave', $new_autosave );
 
 			// wp_update_post() expects escaped array.
-			return wp_update_post( wp_slash( $new_autosave ) );
+			$revision_id = wp_update_post( wp_slash( $new_autosave ) );
+		} else {
+			// Create the new autosave as a special post revision.
+			$revision_id = _wp_put_post_revision( $post_data, true );
+		}
+
+		if ( is_wp_error( $revision_id ) || 0 === $revision_id ) {
+			return $revision_id;
+		}
+
+		// Attached any passed meta values that have revisions enabled.
+		if ( ! empty( $meta ) && $revision_id ) {
```

Comment:
This will always be not empty right? `$revision_id `

Reply:
Right, this will always be set now, I can remove